specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/examples/LISTExtension.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "name": "myMachine",
+    "api-version": "2019-12-12"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "string",
```

Comment:
any meaningful mock value?

Reply:
well... not really at the moment, no... i think these were adding to make some automated process happy. i'll see if we have some reasonable examples to use instead

Reply:
@yungezz ok, updated.